app/views/devise/passwords/edit.html.slim

```diff
@@ -8,7 +8,7 @@ h2 Change your password
     = f.label :password, "New password" 
     br
       - if @minimum_password_length
-    em 
+      em 
       = @minimum_password_length  (characters minimum)
```

Comment:
This line needs to be tabbed in as well Dan, it seems like you're not actually seeing how this stuff renders on the pages.
And it also shows that you haven't really checked out how slim works with it's indentation.